common/src/main/generated/data/bingo/bingo/goal/medium/blue_shield_with_white_flower_charge.json

```diff
@@ -19,11 +19,20 @@
               "minecraft:enchantments": {
                 "levels": {}
               },
+              "minecraft:equippable": {
+                "slot": "offhand",
+                "swappable": false
+              },
+              "minecraft:item_model": "minecraft:shield",
+              "minecraft:item_name": "{\"translate\":\"item.minecraft.shield\"}",
               "minecraft:lore": [],
               "minecraft:max_damage": 336,
               "minecraft:max_stack_size": 1,
               "minecraft:rarity": "common",
-              "minecraft:repair_cost": 0
+              "minecraft:repair_cost": 0,
+              "minecraft:repairable": {
+                "items": "#minecraft:wooden_tool_materials"
+              }
```

Comment:
Maybe it would be a good idea to serialize items without their default components